Project.toml

```diff
@@ -1,26 +1,22 @@
 name = "CPLEX"
 uuid = "a076750e-1247-5638-91d2-ce28b192dca0"
 repo = "https://github.com/jump-dev/CPLEX.jl"
-version = "0.6.6"
+version = "0.7.0"
 
 [deps]
+CEnum = "fa961155-64e5-5f13-b03f-caf6b980ea82"
 Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
-LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 MathOptInterface = "b8f27783-ece8-5eb3-8dc8-9495eed66fee"
-MathProgBase = "fdba3010-5040-5b88-9595-932c9decdf73"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 
 [compat]
-Compat = "2, 3"
+CEnum = "0.3, 0.4"
```

Comment:
To avoid surprises, you could set the lower bound at 0.4 directly

Reply:
`0.3` is fine: https://github.com/JuliaInterop/CEnum.jl/compare/v0.3.0...v0.4.1